src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerMainTest.java

```diff
@@ -134,4 +134,24 @@ void givenCaseCannotBeListedRightAwayAndDirectionsAreBeingMade_thenNoMessagesSen
         assertThat(response.getErrors()).isEmpty();
     }
 
+    @DisplayName("When we have written an adjournment notice and excluded some panel members, add them "
+        + "to the excluded panel members list")
+    @Test
+    void givenPanelMembersExcluded_thenAddPanelMembersToExclusionList() {
+        panelMembersExcludedWithExistingExclusions();

```

Comment:
Please just inline the code for this. There is no need for it to be extracted into a helper method, despite repetition. It is clearer to spell it out in the test it appears.

Reply:
done